onyo/lib/tests/test_commands_unset.py

```diff
@@ -70,24 +60,6 @@ def test_onyo_unset_errors(inventory: Inventory) -> None:
                   message="some subject\n\nAnd a body")
 
 
-@pytest.mark.ui({'yes': True})
-def test_onyo_unset_on_empty_directory(inventory: Inventory) -> None:
```

Comment:
Should probably  add a test above that verifies that this type of call now errors on directories.